util/src/main/java/io/kubernetes/client/util/PagerParams.java

```diff
@@ -0,0 +1,30 @@
+package io.kubernetes.client.util;
+
+public class PagerParams {
+
+  private Integer limit;
+  private String _continue;
```

Comment:
nit: remove underscore.

Reply:
isn't `continue` a reserved key word?

Reply:
good point, but perhaps `continueToken`? underscore is pretty unusual in Java variable names...

Reply:
changed to continueToken.